contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.0.0--2.1.0.sql

```diff
@@ -397,9 +397,9 @@ GRANT EXECUTE ON PROCEDURE sys.sp_column_privileges TO PUBLIC;
 CREATE OR REPLACE VIEW sys.sp_table_privileges_view AS
 -- Will use sp_column_priivleges_view to get information from SELECT, INSERT and REFERENCES (only need permission from 1 column in table)
 SELECT DISTINCT
-CAST(TABLE_QUALIFIER AS sys.sysname) AS TABLE_QUALIFIER,
+CAST(TABLE_QUALIFIER AS sys.sysname) COLLATE sys.database_default AS TABLE_QUALIFIER,
```

Comment:
We should try to avoid changing the previous upgrade scripts. Why do we need to change `2.0.0--2.1.0` upgrade script here?

Reply:
Upgrade path 2.2.0--2.3.0 works fine with above approach. But other upgrade paths, for example, 2.0.0 → 2.1.0, is not working and failing with collation conflict error (during MVU, When we try to upgrade from 1.x to 2.3.0). 
    1. For example, `CREATE OR REPLACE VIEW sys.sp_table_privileges_view throws following error during 2.0.0 --> 2.1.0 upgrade
```
ERROR:  collation mismatch between implicit collations "C" and "bbf_unicode_cp1_ci_as"
            LINE 15: CAST(t2.dbname AS sys.sysname) AS TABLE_QUALIFIER,
                     ^
            HINT:  You can choose the collation by applying the COLLATE clause to one or both expressions.
```

This is happening probably because we upgrade babelfishpg_common extension first and then babelfishpg_tsql which involves re-creation of sys objects. So To resolve this, we updated previous upgrade script only for this branch.

Reply:
For example, `sys.all_objects` is also in `2.1.0--2.2.0`. What happens if a user does mVU from 2.1 to 2.3?

Reply:
This is happening probably because we upgrade babelfishpg_common extension first which will update default collation of sys data types and then babelfishpg_tsql extension which involves execution of multiple minor upgrade path. During `2.0.0--2.1.0`, we are running into issue only for `sys.sp_table_privileges_view` so we have made this change in the definition. 


One more thing to note is that, this object is being recreated during `2.2.0--2.3.0`. So any cx coming to 2.3.0 irrespective of upgrade or fresh install, they will get the same definition.

Reply:
I'm also talking about other objects in this upgrade scripts: `sys.all_objects`, `sys.spt_tablecollations_view`, `sys.objects`, and `sys.spt_tablecollations_view`. Please avoid unnecessary changes.